pages/room/[id].js

```diff
@@ -257,6 +257,7 @@ export default function Room(props) {
                         updateRoomId: room.id,
                         updateRoomUsers: room.userIds,
                         isShown: false,
+                        cards: room.cards,
                       },
                     })
                   }
```

Comment:
It seems like the hide cards button here is being conditionally displayed only when the userId == room.host is not true? Because the hide button is not being displayed for room hosts.